polaris-core/src/main/java/org/apache/polaris/core/persistence/PolarisMetaStoreManager.java

```diff
@@ -391,6 +392,10 @@ ResolvedEntityResult refreshResolvedEntity(
       long entityCatalogId,
       long entityId);
 
+  void flushEventsToPersistence(
```

Comment:
`flush` implies some things that might not be true here, can we just say `writeEvents` again?